robottelo/ui/base.py

```diff
@@ -82,8 +82,14 @@ def navigate_to_entity(self):
         """Perform navigation to main page for specific entity"""
         raise NotImplementedError('Subclasses must implement navigator method')
 
-    def search(self, element_name):
-        """Uses the search box to locate an element from a list of elements."""
+    def search(self, element):
+        """Uses the search box to locate an element from a list of elements.
+
+        :param element: either element name or a tuple, containing element name
+        as a first element and all the rest variables required for element
+        locator.
+        """
+        element_name = element[0] if isinstance(element, tuple) else element
```

Comment:
Or optionally we can implement custom search for docker tag which will collect the elements and pass the tuple to super:
```python
def search(self, name, product_name, repo_name):
    """Use custom search as entity locator value depends on docker tag
    name, product name and repository name.
    """
    return super(DockerTag, self).search((name, product_name, repo_name))
```
So then the call would look like this:
```python
self.dockertag.search('latest', pr.name, repo.name)
# or
self.dockertag.search(name='latest', product_name=pr.name, repo_name=repo.name)
```